controlplane/internal/controller/openshiftassistedcontrolplane_controller_test.go

```diff
@@ -528,6 +540,7 @@ var _ = Describe("Upgrade scenarios", func() {
 		condition := conditions.Get(openshiftAssistedControlPlane, controlplanev1alpha2.UpgradeCompletedCondition)
 		Expect(condition).NotTo(BeNil())
 		Expect(condition.Status).To(Equal(corev1.ConditionFalse))
+		Expect(condition.Message).To(Equal("upgrade to version 4.15.0 in progress\nupgrading cluster"))
```

Comment:
Do we need this message? I think it's superflous (the `upgrading cluster` part)

Reply:
The `upgrading cluster` message is just a stand-in for what CVO would report back and it's going to be appended to the final message we show in our status. Do you mean I should change the message to be more relevant?

Reply:
I mean we can omit it if everything is OK: as we already say "upgrade to version X in progress"  I think there's no need to append "upgrading cluster"
In case of failure, I think adding further details is great

Reply:
Gotcha! That makes sense to show only relevant information and especially not repeating things. Thanks! Removed the upgrading cluster part of the message.